cloud-gateway/src/main/java/com/registry/cloudgateway/dto/CriteriaDTO.java

```diff
@@ -0,0 +1,16 @@
+package com.registry.cloudgateway.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class CriteriaDTO {
+    BigDecimal quantity;
```

Comment:
Does it make sense to use BigDecimal for quantity? Is the reason for this just to make computations easier (require no conversions)?

Reply:
Yes, i did it for computations mainly, quantity should not be like this, but changing it again would require changes again in the DB